back-end/controllers/login.js

```diff
@@ -0,0 +1,62 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const router = express.Router();
+const { queryDB } = require('../helpers/db-helpers');
+
+// Existing user
+router.post('/login', [
+    check('email').exists().isEmail().normalizeEmail(),
+    check('password').exists()
+], async function (req, res, next){
+    try {
+        const { email, password } = req.body;
```

Comment:
Sorry I overlooked this part, but we're not going to be using email anymore and going to use `username`. I think all we have to do is change `email` to `username`

Reply:
Oh yeah my b. Changed